src/schema/me/create_bidder_mutation.js

```diff
@@ -0,0 +1,30 @@
+import { GraphQLNonNull, GraphQLString } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import Bidder from "schema/bidder"
+
+export default mutationWithClientMutationId({
+  name: "Bidder",
+  description: "Create a bidder",
+  inputFields: {
+    sale_id: {
+      type: new GraphQLNonNull(GraphQLString),
+    },
+  },
+  outputFields: {
+    bidder: {
+      type: Bidder.type,
+      resolve: bidder => bidder,
+    },
+  },
+  mutateAndGetPayload: (
+    { sale_id },
+    request,
+    { rootValue: { accessToken, createBidderLoader } }
+  ) => {
+    if (!accessToken) {
+      return new Error("You need to be signed in to perform this action")
```

Comment:
I think this may be a pattern we have been following in MP but I wasn't sure what status the response ends up with when a generic `Error` is thrown.

cc @alloy

Reply:
errors throw in resolvers wont affect the HTTP status code of the response, the data will be `null` and this is added to the `errors` array.

Reply:
Yup 👍

Reply:
Ah you know what, maybe you were thinking of [this](https://github.com/artsy/lewitt-api/blob/7992fa5fb21a1f7a1481b5ed0a2f285181337d64/app/api/graph/root_query.rb#L11)? Yeah I think we should start adopting that convention across the projects. A client can build better upon such codes than messages.

I don’t think we have this in MP yet, though, so it may not be possible out the box to attach metadata with graphql-js like ruby-graphql does.

Reply:
@yuki24 bump ^

Reply:
Yeah `Error` seems a bit too generic and makes it difficult to properly handle it when a dependency also throws e.g. an `Error("connection error")`.